API/Test/test_verify_adding_new_pet.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import requests
+from configuration import BASE_URL
+
+@pytest.fixture
+def pet_data():
+    return {
+        "id": 1,
+        "category": {
+            "id": 0,
+            "name": "Dogs"
+        },
+        "name": "Buddy",
+        "photoUrls": [
+            "http://example.com/photo.jpg"
+        ],
+        "tags": [
+            {
+                "id": 1,
+                "name": "friendly"
+            }
+        ],
+        "status": "available"
+    }
```

Comment:
All fixtures should be stored in conftest.py

Reply:
moved to constants.py